config/initializers/amqp.rb

```diff
@@ -4,6 +4,7 @@
   Sneakers.configure \
     amqp:    "amqp://#{ENV.fetch('RABBIT_USER')}:#{ENV.fetch('RABBIT_PASSWORD')}@#{ENV.fetch('RABBIT_HOST')}:#{ENV.fetch('RABBIT_PORT')}",
     vhost:   ENV.fetch('RABBIT_VHOST'),
+    workers: ENV.fetch('SNEAKERS_WORKERS').to_i,
```

Comment:
I'd suggest doing a parse to`Integer()` here too. 
`Integer ENV.fetch(:my_var)`

If you send `:to_i` to `nil` or some garbage string you'll get a zero back.. not sure how sneakers will handle that & if it will throw an error so its a bit misleading. 

Anything you expect should be a number is probz a good idea to parse with `Integer()`.